custom_components/tuya_local/devices/vibrance_fireplace.yaml

```diff
@@ -1,26 +1,29 @@
 name: Vibrance Fireplace
 products:
   - id: nabdubjo2qqnrxs6
-    manufacturer: Vibrance
-    model: XH-BG-15W/30WZKWIFI-10
+    manufacturer: Modern Ember
+    model: Vibrance
+    model_id: XH-BG-15W/30WZKWIFI-10
 entities:
   - entity: climate
     dps:
-      - id: 1
-        name: switch
-        type: boolean
       - id: 2
         name: temperature
         type: integer
-        unit: C
         range:
           min: 5
           max: 35
+        mapping:
+          - constraint: temperature_unit
+            conditions:
+              - dps_val: C
```

Comment:
The three lines above are conflicting with the next ones below, and do not do anything, so should just be removed.